src/ConcreteOperations/intersection.jl

```diff
@@ -676,21 +676,44 @@ Compute the intersection of two polytopes in vertex representation.
 
 ### Output
 
-A `VPolygon` if both arguments are `VPolygon`s, and a `VPolytope` otherwise.
+A `VPolytope`.
 """
-function intersection(P1::Union{VPolytope{N}, VPolygon{N}},
-                      P2::Union{VPolytope{N}, VPolygon{N}};
+function intersection(P1::VPolytope{N},
+                      P2::VPolytope{N};
                       backend=default_polyhedra_backend(P1, N),
                       prunefunc=removevredundancy!) where {N<:Real}
-    Q1 = polyhedron(convert(VPolytope, P1); backend=backend)
-    Q2 = polyhedron(convert(VPolytope, P2); backend=backend)
+    Q1 = polyhedron(P1; backend=backend)
+    Q2 = polyhedron(P2; backend=backend)
     Pint = Polyhedra.intersect(Q1, Q2)
-    prunefunc(Pint)
```

Comment:
Now the `prunefunc` parameter is not used anymore. Why?

Reply:
i see, it wasn't intentional.